models/stg_hubspot__owner_team.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_team_enabled'])) }}
```

Comment:
Same comment as source -- should this be dependent on `hubspot_owner_enabled` at all?